.github/workflows/check-format.yml

```diff
@@ -0,0 +1,16 @@
+name: Check Code Formatting
+
+on:
+  pull_request:
+    # branches:
+    # - main
```

Comment:
I'll uncomment this before merging to main. This is currently diffed against #95.